### PR TITLE
Test fix Kafka Performance test batch

### PR DESCRIPTION
**Pull request by Abacn**
**Please** add a meaningful description for your change here

------------------------

Thank you for your contribution! Follow this checklist to help us incorporate your contribution quickly and easily:

 - [ ] [**Choose reviewer(s)**](https://beam.apache.org/contribute/#make-your-change) and mention them in a comment (`R: @username`).
 - [ ] Mention the appropriate issue in your description (for example: `addresses #123`), if applicable. This will automatically add a link to the pull request in the issue. If you would like the issue to automatically close on merging the pull request, comment `fixes #<ISSUE NUMBER>` instead.
 - [ ] Update `CHANGES.md` with noteworthy changes.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

See the [Contributor Guide](https://beam.apache.org/contribute) for more tips on [how to make review process smoother](https://beam.apache.org/contribute/get-started-contributing/#make-the-reviewers-job-easier).

To check the build health, please visit [https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md](https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md)

GitHub Actions Tests Status (on master branch)
------------------------------------------------------------------------------------------------
[Build python source distribution and wheels](https://github.com/apache/beam/actions?query=workflow%3A%22Build+python+source+distribution+and+wheels%22+branch%3Amaster+event%3Aschedule)
[Python tests](https://github.com/apache/beam/actions?query=workflow%3A%22Python+Tests%22+branch%3Amaster+event%3Aschedule)
[Java tests](https://github.com/apache/beam/actions?query=workflow%3A%22Java+Tests%22+branch%3Amaster+event%3Aschedule)
[Go tests](https://github.com/apache/beam/actions?query=workflow%3A%22Go+tests%22+branch%3Amaster+event%3Aschedule)

See [CI.md](https://github.com/apache/beam/blob/master/CI.md) for more information about GitHub Actions CI.

### CODE REVIEW

**Comment by Abacn** [comment]
Run Java KafkaIO Performance Test

**Comment by Abacn** [comment]
Run Java KafkaIO Performance Test

**Comment by Abacn** [comment]
Run Java KafkaIO Performance Test

**Comment by Abacn** [comment]
Run Java KafkaIO Performance Test

**Comment by Abacn** [comment]
Run Java KafkaIO Performance Test

**Comment by Abacn** [comment]
Run Java KafkaIO Performance Test

**Comment by Abacn** [comment]
There are three problems:
1. K8s instance used for testing running out of memory and causing kafka workload fail to initialize. Cleaned up in the backend.
2. testKafkaIOReadsAndWritesCorrectlyInBatch trying to write/read non-exist kafka topic. Fixed in config. Needs to run seed job to take effect.
3. Once the first two issues resolved, found that the test does not end in time. Bumped timeout.

File: .test-infra/kubernetes/kafka-cluster/05-kafka/configmap-config.yaml [hunk 1]
```diff
@@ -35,7 +35,7 @@ data:
     echo "Applying runtime configuration using confluentinc/cp-kafka:5.0.1"
     kafka-topics --zookeeper zookeeper:2181 --create --if-not-exists --force --topic beam --partitions 1 --replication-factor 3
     kafka-configs --zookeeper zookeeper:2181 --entity-type topics --entity-name beam --describe
-    kafka-topics --zookeeper zookeeper:2181 --create --if-not-exists --force --topic beam-runnerv2 --partitions 1 --replication-factor 3
```
Comment (Abacn) [inline .test-infra/kubernetes/kafka-cluster/05-kafka/configmap-config.yaml:38]: This topic is no longer used as the duplicate test removed in #23027

**Comment by Abacn** [comment]
Run Java KafkaIO Performance Test

**Comment by Abacn** [comment]
Test now (correctly) time out. Running seed job should resolve.

**Comment by Abacn** [comment]
R: @johnjcasey

**Comment by johnjcasey** [comment]
run seed job

**Comment by johnjcasey** [comment]
Run Java KafkaIO Performance Test

**Comment by johnjcasey** [comment]
Run Java KafkaIO Performance Test

**Comment by Abacn** [comment]
The test configuration change still does not take effect. From log it shows "--kafkaTopic=beam","--readTimeout=900" for batch read job. If it takes effect this should be "--kafkaTopic=beam-batch","--readTimeout=1800"

**Comment by johnjcasey** [comment]
Merging this despite failing performance test. The test never worked, and we are merging these changes in to try and fix it

**Comment by Abacn** [comment]
Pipeline now runs, though get assertion error: "Expected: "0f12c27c9a7672e14775594be66cad9a". but: was "966684bbc4d071c54466678005c86eec"

**Pull request merged**